test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,209 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test functionality on the cluster with different values of the --smp parameter on the nodes.
+"""
+import logging
+import sys
+
+import pytest
+
+sys.path.insert(0, sys.path[0] + "/test/cql-pytest")
+import nodetool
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables, TextType, Column
+from test.pylib.util import unique_name
+
+logger = logging.getLogger(__name__)
+CONFIG = {"endpoint_snitch": "GossipingPropertyFileSnitch"}
+
+
+# Checks a cluster boot/operations in multi-dc environment
+@pytest.mark.asyncio
+@pytest.mark.skip(reason="Too heavy for CI, but OK for local run")
+async def test_multidc(request: pytest.FixtureRequest, manager: ManagerClient) -> None:
+    logger.info("Creating a new node")
```

Comment:
It looks like you are adding now "a new node", but 30 of them. 
Is there any benefit to adding 30 and not 10 or 3?

Reply:
This test is copypaste. I do believe the idea of this test is to check that ScyllaDB will be able to handle a large amount of DC. Each node in this test is added to its own DC, so the test generally checks the ability to work with 30 DC.

Reply:
copypaste from what?
If the intention is to check a large number of DCs, this should be specified in the test's comment, and perhaps even the test's name (e.g., call it many_dc instead of multi_dc). But then you need to ask yourself, is 30 really many? What kind of datastructure might we have in Scylla which can deal with 10 DCs fine, but 30 DCs is already "many"?

Reply:
5 DCs is plenty, and atypical

Reply:
One of our largest customers uses 5 DCs...

Reply:
Changed to 5.